src/seedu/addressbook/AddressBook.java

```diff
@@ -1221,7 +1234,7 @@ private static String getUsageInfoForExitCommand() {
 	 * @return string without the sign
 	 */
 	private static String removePrefixSign(String s, String sign) {
-		return s.replace(sign, "");
+		return removePrefix(s, sign);
```

Comment:
this is a nice way to implement the generalization :+1:
you have ensured that the rest of the code is not disturbed. quite a unique attempt :-)